plugins/telink-dfu/fu-telink-dfu-ble-device.c

```diff
@@ -90,16 +90,17 @@ fu_telink_dfu_ble_device_ota_start(FuTelinkDfuBleDevice *self, GError **error)
 static gboolean
 fu_telink_dfu_ble_device_ota_stop(FuTelinkDfuBleDevice *self, guint number_chunks, GError **error)
 {
-	guint8 pkt_stop_data[4] = {0};
+	g_autoptr(FuStructTelinkDfuEndCheck) end_check = fu_struct_telink_dfu_end_check_new();
```

Comment:
these rustgen-specified structs normally have a st_ prefis, e.g. `st_end_check`

Reply:
resolved in e84eeda9e6af461945b34f32c7376aea3f5b6bea